test/unit-test/FreeRTOS_DNS/FreeRTOS_DNS_utest.c

```diff
@@ -1040,6 +1040,8 @@ void test_FreeRTOS_getaddrinfo_a_IPv6Random_BindFailWithDot( void )
     struct xSOCKET xDNSSocket;
     NetworkEndPoint_t xEndPoint[ 5 ];
 
+    xDNS_IP_Preference = xPreferenceIPv6;
```

Comment:
We have to manually set this because xDNS_IP_Preference is always xPreferenceIPv4?